pkg/kv/kvserver/kvflowcontrol/rac2/range_controller_test.go

```diff
@@ -107,8 +118,8 @@ func (r *testingRCRange) admit(
 	for _, replica := range r.mu.r.replicaSet {
 		if replica.desc.StoreID == storeID {
 			replica := replica
-			replica.info.Admitted[AdmissionToRaftPriority(pri)] = toIndex
-			replica.info.Term = term
+			replica.av.Admitted[AdmissionToRaftPriority(pri)] = toIndex
+			replica.av.Term = term
```

Comment:
What happens with the priorities != `pri` when `Term` goes up? Should we rather pass the whole `AdmittedVector` in and update it wholesale?